src/test/java/uk/gov/hmcts/reform/bulkscanprocessor/helper/EnvelopeCreator.java

```diff
@@ -63,11 +64,11 @@ public static EnvelopeResponse envelopeResponse() throws Exception {
         return ImmutableList.of(envelope());
     }
 
-    public static Envelope envelope() throws Exception {
+    public static Envelope envelope() {
```

Comment:
All the changes in this class just make it possible to use this (and other) methods in streams (checked exception is not thrown).
Not much to see here ;)